src/main/java/frc/robot/Constants.java

```diff
@@ -41,7 +41,7 @@
     public static final int kDriveBaseCurrentLimit = 40;
 
     //Controller constants
-    public static final int kDBJoystickPort = 0;
+    //public static final int kDBJoystickPort = 0;
```

Comment:
Can remove this line if commenting it out didn't break the build. Seems like a duplicate of kPortNumber